Sources/BraintreePayPalNativeCheckout/BTPayPalNativeTokenizationClient.swift

```diff
@@ -18,7 +18,7 @@ class BTPayPalNativeTokenizationClient {
 
     func tokenize(
         request: BTPayPalRequest,
-        returnURL: String,
+        approvalData: ApprovalData? = nil,
```

Comment:
With approval data being optional here what happens if it doesn't exist and isn't passed into the parameters below? Previously the `returnURL` was required. Should we throw an error if we don't get any approval data?